launcher/package/rpm/MultiMC5.spec

```diff
@@ -1,14 +1,20 @@
 Name:           MultiMC5
 Version:        1.4
-Release:        3%{?dist}
+Release:        4%{?dist}
 Summary:        A local install wrapper for MultiMC
 
 License:        ASL 2.0
 URL:            https://multimc.org
-BuildArch:      x86_64
+ExclusiveArch:  %{ix86} x86_64
+
+BuildRequires:  desktop-file-utils
+BuildRequires:  libappstream-glib
+Requires:       zenity %{?suse_version:lib}qt5-qtbase wget xrandr
```

Comment:
conditional logic requires `libqt5-qtbase` on Suse systems and `qt5-qtbase` on any other system